src/main/java/net/mrscauthd/beyond_earth/events/Gravity.java

```diff
@@ -9,11 +9,13 @@
 
 public class Gravity {
     public static void Gravity(LivingEntity entity, GravityType type, Level world) {
-        double moon = 0.03;
-        double mars = 0.04;
-        double mercury = 0.03;
-        double venus = 0.04;
-        double orbit = 0.02;
+        if (Methods.isWorld(world, Methods.overworld)) return;
```

Comment:
> that is a so small thing that we not need this

Without this, for every item that's on the ground that's not in a space world, it has to check if the current world against 6 different ones, at least 20 times per second.